.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Should we assure `CI` workflow succeeds before running this `Release` workflow? With current changes, the `CI` workflow and `Release` workflow will run in parallel.

Here's an example of how to do so:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow

Reply:
Ideally i wanted to run the workflows first, but from what i found is that you can only filter on workflow_run by branch, you can't filter by tag which means it wont work in this case. Also i looked at our other repositories and we seem to do it like this so i opted for this

Reply:
If we are doing it for other repos let's go for this. 

I see an opportunity to centralize the workflow in a repo and just re-use the workflow across repos so we avoid having this conversations everytime we create a new repo :)